test/unit/org/apache/cassandra/config/DatabaseDescriptorRefTest.java

```diff
@@ -54,6 +54,8 @@
 public class DatabaseDescriptorRefTest
 {
     static final String[] validClasses = {
+    "org.apache.cassandra.config.CassandraRelevantProperties",
+    "org.apache.cassandra.config.CassandraRelevantProperties$PropertyConverter",
```

Comment:
Those should be moved down I think, I see the classes are listed in alphabetic order